packages/blueprints/src/Blueprints.js

```diff
@@ -180,14 +180,14 @@ class Blueprints {
               (newItem.metaData.type !== existingItem.metaData.type ||
                 newItem.metaData.title !== existingItem.metaData.title ||
                 newItem.metaData.description !==
-                  existingItem.metaData.description)) ||
+                  existingItem.metaData.description ||
+                newItem.metaData.width !== existingItem.metaData.width ||
+                newItem.metaData.height !== existingItem.metaData.height)) ||
             (newItem.metaData &&
               newItem.metaData.type === 'text' &&
               existingItem.metaData &&
               existingItem.metaData.type === 'text' &&
-              (newItem.metaData.width !== existingItem.metaData.width ||
-                newItem.metaData.height !== existingItem.metaData.height ||
-                newItem.metaData.html !== existingItem.metaData.html ||
```

Comment:
Why this was deleted?

Reply:
I moved it up for the general items check since width and height is not only relevant to text items but for all types